packages/ui/src/views/labels/components/label-form-color-and-name-group.tsx

```diff
@@ -1,8 +1,7 @@
 import { FC } from 'react'
-import { UseFormRegister } from 'react-hook-form'
 
-import { Button, Icon, Input, Select } from '@/components'
-import { ColorsEnum, CreateLabelFormFields, TranslationStore } from '@/views'
+import { Button, Icon, Input, InputProps, Select, SelectRootProps } from '@/components'
+import { cn, ColorsEnum, TranslationStore } from '@/views'
```

Comment:
Has `cn` been accidentally added to the views? This should only be importable from `@utils`.